VMEncryption/main/handle.py

```diff
@@ -138,7 +138,7 @@ def disable_encryption():
             if crypt_item.dev_path.startswith("/dev/sd"):
                 logger.log('Updating crypt item entry to use mapper name')
                 logger.log('Device name before update: {0}'.format(crypt_item.dev_path))
-                crypt_item.dev_path = disk_util.query_dev_id_path_by_sdx_path(crypt_item.dev_path)
+                crypt_item.dev_path = disk_util.get_persistent_path_by_sdx_path(crypt_item.dev_path)
```

Comment:
Fallback to dev by-id path incase persistent paths are missing?

Reply:
I don't know enough about by-id to add that here just yet.
Perhaps we can do it later after a little more research.